### PR TITLE
travis: upload coverage

### PR DESCRIPTION
**Pull request by TomasVotruba**
I think this is missing to satisfy scrutinizer

### CODE REVIEW

**Comment by Ocramius** [comment]
@TomasVotruba nope - currently hangs on phpcs for some reason (unclear)

File: .travis.yml [hunk 1]
```diff
@@ -6,9 +6,13 @@ php:
 
 before_script:
   - composer update --prefer-source
+  - if [[ $TRAVIS_PHP_VERSION = '7.0' ]]; then PHPUNIT_FLAGS="--coverage-clover coverage.xml"; fi
```
Comment (Ocramius) [inline .travis.yml]: Not needed - we always run with 7.0

**Comment by TomasVotruba** [comment]
@Ocramius Where exactly? I'd like to fix these badges :)

**Comment by Ocramius** [comment]
https://scrutinizer-ci.com/g/Ocramius/PSR7Session/inspections/2eae85d9-d4e1-49f5-a489-d01c3c413934

20d1f3

**Comment by TomasVotruba** [comment]
Looks like Scrutinizer's version of phpcs is outdated.

I tried with phpcs 2.3.4 and timeoutes too on some Squiz sniff.

With version 2.5 all passes well. Probably it has bug fixed.

Closing in favor of #41

**Pull request closed without merging**